tests/fips/openjdk/openjdk_fips.pm

```diff
@@ -0,0 +1,70 @@
+# SUSE's openjdk fips tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: openjdk expect
+#
+# Summary: FIPS: openjdk
+#          Jira feature: SLE-21206
+#          FIPS 140-3: make OpenJDK be able to use the NSS certified crypto
+#
```

Comment:
Please describe the `Case 1` here too since I see you have mentioned Case2 in `openjdk_ssh.pm `case